CompuCell3D/core/CompuCell3D/steppables/PDESolvers/OpenCL/DiffusionSolverFE_OpenCL.cpp

```diff
@@ -740,9 +740,23 @@ void DiffusionSolverFE_OpenCL::initImpl() {
 
     //    "lib/CompuCell3DSteppables/OpenCL/DiffusionKernel.cl" };
 
+//    const char *kernelSource[] = {
+//            (string("lib/site-packages/cc3d/cpp/CompuCell3DSteppables/OpenCL/GPUSolverParams.h")).c_str(),
+//            (string("lib/site-packages/cc3d/cpp/CompuCell3DSteppables/OpenCL/DiffusionKernel.cl")).c_str()};
+//
+
+    string cc3d_open_cl_solvers_dir = string(getenv("CC3D_OPENCL_SOLVERS_DIR"));

```

Comment:
Should you handle the case where `getenv` returns null? Also, should you abort the diffusion when the directory does not exist?

Reply:
Fixed: Added handling of null pointer but when directory does not exist we get notification from OpenCL that it cannot find required file which is good enough for now

Reply:
Nice!